packages/bson-transpilers/test/yaml/partial.yaml

```diff
@@ -356,6 +386,7 @@ tests:
           java: 429496729601L -
           csharp: 429496729601L -
           shell: new NumberLong(429496729601) -
+          php: 429496729601 -
```

Comment:
Why is this using a `-` operator? Ignoring the fact that other languages are using it, as I don't think we should be blindly copying things without understanding them.

Docs for [Long.compare](https://mongodb.github.io/node-mongodb-native/api-bson-generated/long.html#compare) in the Node driver show is equivalent to PHP's `<=>` operator in returning -1, 0, or 1.

Reply:
This is a partial test and this is only a part of syntax construction. Processing templates are "LongCompareTemplate" and "LongCompareArgsTemplate". In this case only "LongCompareTemplate" works because we don't have the second item to compare with.

Reply:
I still don't understand why we're using subtraction instead of the comparison operator (`<=>`). That seems like a mistake.